### PR TITLE
Default path for the configuration file

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,9 +145,13 @@ $ add-react-component --no-index Example
...
 generators, technology templates and filenames. Look into `config.js` to find out what cat be setted.
 
+If you store your configuration file by `.add-component/config.js` path, you do not need any additional parameter. Just
+Run the command as usual.
+
+If you want your configuration file to have another name or be in another folder, tell the command where it is:
 ```
 # Run with configuration
-$ add-react-component --config .add-component/config.js Example
+$ add-react-component --config congifs/addcomponent-config.js Example
```
Comment (hanford) [inline README.md]: i think you mean configs

**Review by hanford — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (varya) [reply README.md]: My bad :-) Fixed.

**Comment by hanford** [comment]
dope!

**Comment by hanford** [comment]
both of your changes are live in v2.5.0! Thanks again 🤗

**Comment by varya** [comment]
🎉 Thanks!

**Pull request merged**